src/ostorlab/runtimes/lite_local/agent_runtime.py

```diff
@@ -348,6 +348,10 @@ def create_agent_service(
             or self.agent.container_image.replace(":", "_").replace(".", "")
             + "_"
             + self.runtime_name
+            + "_"
+            + str(uuid.uuid4())[
```

Comment:
This will cause problem, there is a limit on the service name size. The value should be incremental with .1 .2. 3.

Reply:
Also, this updating on the lite_local runtime, it should be applied to all runtimes!